tests/operator_test.go

```diff
@@ -671,11 +676,73 @@ var _ = Describe("ALL Operator tests", func() {
 		})
 
 		var _ = Describe("Alert tests", func() {
+			var numAddedStorageClasses int
 			f := framework.NewFramework("alert-tests")
 
+			AfterEach(func() {
+				By("Delete unknown storage classes")
+				for i := 0; i < numAddedStorageClasses; i++ {
+					name := fmt.Sprintf("unknown-sc-%d", i)
+					_, err := f.K8sClient.StorageV1().StorageClasses().Get(context.TODO(), name, metav1.GetOptions{})
```

Comment:
This isn't necessarily wrong, but there really is no reason to get the storage class. Simply delete it, and ignore not found errors. There is no guarantee that in between this get, and the delete, the storage class didn't get deleted by something else.

Reply:
Yeah I see what you mean, if it's fine with you, we can merge as is

Reply:
Yeah, its fine with me, we have that pattern all over the place, just noting that it is not really needed.